packages/manager/apps/hub/src/index.routes.js

```diff
@@ -0,0 +1,7 @@
+export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
```

Comment:
Shouldn't be named as `routing.js` instead of `index.routes.js`?

Reply:
No official convention about this but I don't see any issue renaming it